### ISSUE TITLE
Code of Conduct

### ISSUE THREAD

**Issue by amtwo** [sqlcollaborative/DataSaturdays#47]
We need a Code of Conduct (or Codes of Conduct) to support:
* Contributing to this repo
* Participants at events (Speakers, sponsors, organizers, etc)
    * Encompassing both in-person & virtual events

# GitHub contributing Code of Conduct
This seems pretty straightforward in using the same Code of Conduct as other sqlcollaborative projects.

# Event Code of Conduct
In #27 @andrekamman [mentions](https://github.com/sqlcollaborative/DataSaturdays/issues/27#issuecomment-748904855) the [CodeOfConduct.com](https://github.com/confcodeofconduct/confcodeofconduct.com) code as a base, which helps tackle internationalization/translation.

Events should have a "base" code of conduct (like above), plus a "supplement" which has event-specific details.

## Requirements:
* The event-specific supplement should include who to report violations to, and how/where to contact them. 
    * Reporting/assistance should be a _direct, unambiguous_ path from the Code of Conduct. (ie, a monitored email address, a front desk person at an in-person event, etc)
    * Provide guidance to event organizers on eliminating single points of failure for individuals seeking help in reporting problems. Event organizers are often busy & not able to provide a fast initial response
* The event Code of which is featured prominently & is easy to find. When an attendee is "in crisis" as the victim of harassment, it should not be difficult to find the Code of Conduct (and hence the path to reporting/assistance)

We should additionally provide resources for event organizers so that they feel equipped to provide assistance & response.

**Comment by way0utwest** [comment]
I agree with the above, and I think a CoC/AHP ought to be one of the few rules we implement.

**Comment by tameraclark** [comment]
Microsoft is requiring a CoC to be part of the Data Community they are setting up for user groups. It was just a requirement to be available on the MeetUp group. I resued the one we created for our last #SQLSatNash and posted it on our site. As long as you make the requirement verifiable this should be easy.  https://nashbi.sqlugs.com/code-of-conduct/

One requirement that I would like to see and maybe it's just a recommendation. 1) HOW to report something and 2) that organizers plan for having a male & female available to talk to an attendee should there be an issue.  I am not 100% certain I would feel like expressing my concerns to a guy.

**Comment by way0utwest** [comment]
Thanks, I appreciate seeing this link. Getting a few CoCs is important.

**Comment by tameraclark** [comment]
> Thanks, I appreciate seeing this link. Getting a few CoCs is important.

I reviewed several before working on this. I also added based on the feedback of doing my session "The community speaks". Getting real feedback from people who were actually attending events was very helpful!
